src/nodes/display/BleachBypassNode.js

```diff
@@ -0,0 +1,27 @@
+import { addNodeElement, float, tslFn, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { min, max, dot, mix } from '../math/MathNode.js';
+
+export const bleach = /*@__PURE__*/ tslFn( ( [ color, opacity = 1 ] ) => {
```

Comment:
Please do not use `{}` for the parameters. If you do so, the pass does not work when doing this:
```js
const scenePass = pass( scene, camera );
const scenePassColor = scenePass.getTextureNode();

postProcessing.outputNode = scenePassColor.bleach();
```